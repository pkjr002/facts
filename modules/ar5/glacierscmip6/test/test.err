Traceback (most recent call last):
  File "ar5_project_glacierscmip6.py", line 244, in <module>
    ar5_project_glacierscmip6(args.seed, args.pyear_start, args.pyear_end, args.pyear_step, args.nmsamps, args.ntsamps, args.nsamps, args.pipeline_id)
  File "ar5_project_glacierscmip6.py", line 103, in ar5_project_glacierscmip6
    raise ProjectionError('number of realisations '+\
__main__.ProjectionError: number of realisations must be a multiple of number of glacier methods
mv: cannot stat ‘*globalsl*’: No such file or directory
mv: cannot stat ‘*localsl*’: No such file or directory
