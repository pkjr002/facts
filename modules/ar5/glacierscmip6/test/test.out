Initiating test in /scratch/rk509/test.1666667491...
Extracting data files...
Executing workflow...
python ar5_preprocess_glacierscmip6.py --pipeline_id glaciers-ar5-glacierscmip6-ssp585 --scenario ssp585 --baseyear 2005
python ar5_fit_glacierscmip6.py --pipeline_id glaciers-ar5-glacierscmip6-ssp585
python ar5_project_glacierscmip6.py --pipeline_id glaciers-ar5-glacierscmip6-ssp585 --nsamps 20000 --pyear_start 2020 --pyear_end 2100 --pyear_step 10 --seed 4321
python ar5_postprocess_glacierscmip6.py --pipeline_id glaciers-ar5-glacierscmip6-ssp585 --locationfile test/location_list.lst
Collecting output files...
